src/csf/transformCsf.ts

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable no-underscore-dangle */
-import { loadCsf } from '@storybook/csf-tools';
+import { toId, storyNameFromExport, combineTags } from '@storybook/csf';
```

Comment:
maybe if you import these from `storybook/internal/common` the type conflict goes away?

Reply:
unfortunately not because these are not exported by storybook/internal/common